js/client/modules/@arangodb/testutils/test-utils.js

```diff
@@ -277,7 +290,7 @@ function splitBuckets (options, cases) {
 function doOnePathInner (path) {
   return _.filter(fs.list(makePathUnix(path)),
                   function (p) {
-                    return (p.substr(-3) === '.js') || (p.substr(-3) === '.rb');;
+                    return (p.substr(-3) === '.js');
```

Comment:
```suggestion
                    return p.endsWith('.js');
```

Reply:
this will make backports more complicated, since its not supported with old V8.